src/client/eppo-client-with-bandits.spec.ts

```diff
@@ -167,6 +171,34 @@ describe('EppoClient Bandits E2E test', () => {
       expect(banditEvent.actionNumericAttributes).toStrictEqual({ brand_affinity: -1 });
       expect(banditEvent.actionCategoricalAttributes).toStrictEqual({ loyalty_tier: 'bronze' });
       expect(banditEvent.metaData?.obfuscated).toBe(false);
+
+      expect(banditEvent.evaluationDetails.configFetchedAt).toBeTruthy();
+      expect(typeof banditEvent.evaluationDetails.configFetchedAt).toBe('string');
+      const expectedEvaluationDetails: IFlagEvaluationDetails = {
+        configFetchedAt: expect.any(String),
+        configPublishedAt: '2024-04-17T19:40:53.716Z',
+        environmentName: 'Test',
+        flagEvaluationCode: 'MATCH',
+        flagEvaluationDescription:
+          'bob belongs to the range of traffic assigned to "banner_bandit" defined in allocation "training".',
+        matchedAllocation: {
+          allocationEvaluationCode: AllocationEvaluationCode.MATCH,
+          key: 'training',
+          orderPosition: 2,
+        },
+        matchedRule: null,
+        unevaluatedAllocations: [],
+        unmatchedAllocations: [
+          {
+            allocationEvaluationCode: AllocationEvaluationCode.TRAFFIC_EXPOSURE_MISS,
+            key: 'analysis',
+            orderPosition: 1,
+          },
+        ],
+        variationKey: 'banner_bandit',
+        variationValue: 'banner_bandit',
```

Comment:
I'd expect fields like `action` and `banditKey` to be populated here

Reply:
added banditKey and banditAction